### PR TITLE
new python lib pafy init version 0.3.74

### PR DESCRIPTION
**Pull request by odi**
Add new python lib: https://github.com/mps-youtube/pafy

### CODE REVIEW

File: pkgs/top-level/python-packages.nix [hunk 1]
```diff
@@ -16790,4 +16790,24 @@ let
...
+
+    src = pkgs.fetchurl {
+      url = "https://pypi.python.org/packages/source/p/pafy/pafy-0.3.74.tar.gz";
+      md5 = "fbf0e7f85914eaf35f87837232eec09c";
+    };
+
+    propagatedBuildInputs = with self; [ youtube-dl ];
+
+    meta = with stdenv.lib; {
+      description = "Python library to download YouTube content and retrieve metadata.";
```
Comment (jagajaga) [inline pkgs/top-level/python-packages.nix]: Change description to `A library to download YouTube content and retrieve metadata`
Please notice that there is no dot in the end.
Thank you!

File: pkgs/top-level/python-packages.nix [hunk 2]
```diff
@@ -16790,4 +16790,24 @@ let
       maintainers = with maintainers; [ kamilchm ];
     };
   };
+
+  pafy = buildPythonPackage rec {
+    name = "pafy-${version}";
+    version = "0.3.74";
+
+    src = pkgs.fetchurl {
+      url = "https://pypi.python.org/packages/source/p/pafy/pafy-0.3.74.tar.gz";
```
Comment (nckx) [inline pkgs/top-level/python-packages.nix]: Might as well use `.../${name}.tar.gz` here.

**Comment by nckx** [comment]
@odi: could you squash all commits into one, rebasing against current master?

**Comment by nckx** [comment]
Thanks!

**Pull request merged**